ts/screens/authentication/cie/CieCardReaderScreen.tsx

```diff
@@ -249,9 +295,11 @@ class CieCardReaderScreen extends React.PureComponent<Props, State> {
         <Content bounces={false} noPadded={true}>
           <Text style={styles.padded}>{this.state.subtitle}</Text>
           <CieReadingCardAnimation readingState={this.state.readingState} />
-          {this.state.content && (
-            <Text style={styles.padded}>{this.state.content}</Text>
-          )}
+          {this.state.isScreenReaderEnabled &&
+            this.accessibilityViewAlternativeFocus}
```

Comment:
On android: If I scroll the elements with the screen reader this element is discovered as empty such as on the screen there is something not explained, can we avoid the use of an empty element?

Reply:
It is used to read the card reading message when it changes:
- focus on message to read <Text>
- the same <Text> change the text
- focus on that dummy element
- focus on <Text> again to read the updated message

I don't know how to avoid it.
Any ideas?

Reply:
I think in this case, if we want to avoid to bounce up and down the focus the only way is to use announceForAccessibility() announcing the text of the content instead of focusing on it?

Reply:
it sounds good!
I'm going to try it....

Reply:
it works! https://github.com/pagopa/io-app/pull/2060/commits/130b6e9f51c77d0dd72e3b0c02d632f6eca54ba0